common/session.py

```diff
@@ -57,8 +58,40 @@ def __init__(self, login=True, add_provider=True, request = None):
 
     def __get_web_driver__(self):
 
-        self.logger.info("Using Browser: %s", self.BROWSER)
-        self.web_driver = getattr(webdriver,self.BROWSER)()
+        if self.BROWSER == "Firefox":
+            self.logger.info("Using Browser: %s", self.BROWSER)
+            profile = webdriver.FirefoxProfile()
+            # enable auto download
+            # http://stackoverflow.com/questions/24852709/how-do-i-automatically-download-files-from-a-pop-up-dialog-using-selenium-python
+            profile.set_preference("browser.download.folderList", 2)
+            profile.set_preference("browser.download.manager.showWhenStarting", False)
+            profile.set_preference("browser.download.dir", '~/Downloads/')
```

Comment:
I question as to whether ~/Downloads will resolve properly.
Perhaps consider using something similar to the following, where the path resolves to the HOME environment variable:

"{}{}".format(os.getenv("HOME"),'/Downloads')